dist/dotty.rb

```diff
@@ -0,0 +1,32 @@
+class Dotty < Formula
+  desc "Experimental Scala Compiler"
+  homepage "http://dotty.epfl.ch/"
+  url "http://dotty.epfl.ch/dotty/dotty-0.1.1-bin-SNAPSHOT.tar.gz"
```

Comment:
Thanks a lot for the tip @olafurpg , I think we should automate the publishing to Github.